src/Autocomplete/Autocomplete.test.jsx

```diff
@@ -44,6 +44,23 @@ describe('<Autocomplete />', () => {
     })
   })
 
+  describe('with a name', () => {
+    it('sets the input field name', () => {
+      const wrapper = mount(
+        <Autocomplete name='name' />
+      ).find(TextField)
+      expect(wrapper.props().name).toBe('name')
+    })
+
+    it('does not add a name field', () => {
+      const wrapper = mount(
+        <Autocomplete />
+      ).find(TextField)
+      console.log(wrapper.props().name)
```

Comment:
Oops

Reply:
Removed and force pushed.

![giphy (5)](https://user-images.githubusercontent.com/127084/55367948-3f016c00-553b-11e9-93c3-549f91d8db66.gif)